Parsers/ASimRegistryEvent/test/TrendMicro_VisionOne_vimRegistryEvent_DataTest.csv

```diff
@@ -0,0 +1,9 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 1 records (0.44%) for field [DvcHostname] of type [Hostname]: [""QA_test""] (Schema:RegistryEvent)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 1 records (0.44%) for field [Dvc] of type [Hostname]: [""QA_test""] (Schema:RegistryEvent)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 229 records (100.0%) for field [EventProduct] of type [Enumerated]: [""Vision One""] (Schema:RegistryEvent)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 229 records (100.0%) for field [EventType] of type [Enumerated]: [""RegistryValueSet""] (Schema:RegistryEvent)"
```

Comment:
Please update EventProduct and EventVendor values in Tester csv

Reply:
Done